drivers/scheduler/k8s/k8s.go

```diff
@@ -4901,6 +4904,10 @@ func (k *K8s) createRbacObjects(
 		return clusterrole, nil
 	} else if obj, ok := spec.(*rbacv1.ClusterRoleBinding); ok {
 		obj.Namespace = ns.Name
+		for i := range obj.Subjects {
+			// since everything in a spec is in the same namespace in cluster, we can set here:
+			obj.Subjects[i].Namespace = ns.Name
+		}
```

Comment:
There is no namespace associated with cluster level objects like `ClusterRoleBinding`

Reply:
There isn't. But the subjects _can_ be namespaced in which case this assignment needs to be done. Otherwise, the subjects end up in the default namespacce.
Refer: https://qubitpi.github.io/kubernetes-website/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1/#:~:text=subjects.namespace%20(string,report%20an%20error.

Reply:
But it says

```
If the object kind is non-namespace, such as "User" or "Group", and this value is not empty the Authorizer should report an error.

```

Reply:
Well, in this case, the object is `ServiceAccount` which is namespaced, so that needs to be set (https://github.com/portworx/torpedo/blob/57e77d7d1dca4057bf92c2697caf09edc2e6b75a/drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/operator.yaml#L355).

But I have modified the code to behave properly.